cmd/consensus/main.go

```diff
@@ -143,6 +144,8 @@ func main() {
 			flags.StringVar(&accessAddress, "access-address", "", "the address of an access node")
 			flags.StringVar(&secureAccessNodeID, "secure-access-node-id", "", "the node ID of the secure access GRPC server")
 			flags.BoolVar(&insecureAccessAPI, "insecure-access-api", true, "required if insecure GRPC connection should be used")
+			flags.DurationVar(&dkgControllerConfig.BaseStartDelay, "dkg-controller-base-start-delay", dkgmodule.DefaultBaseStartDelay, "used to define the range for jitter prior to DKG start")
```

Comment:
better to provide an example. For instance, `500ms`, as is the default value

Reply:
https://github.com/onflow/flow-go/pull/1303/commits/7ffc1c1ec0fb586a42e6af89d157e69e93c479d2